compiler/rustc_expand/src/build.rs

```diff
@@ -152,6 +152,25 @@ impl<'a> ExtCtxt<'a> {
         ast::Stmt { id: ast::DUMMY_NODE_ID, span: expr.span, kind: ast::StmtKind::Expr(expr) }
     }
 
+    pub fn stmt_let_pat(
+        &self,
+        sp: Span,
+        _mutbl: bool,
```

Comment:
fix: seems like this parameter should just be removed, then?  It'll be part of the `Pat` in any case that needs it.

(Conveniently all the other parameters have non-primitive types, so consistency with `stmt_let` doesn't seem worth it to me.)

Reply:
Good catch. I think I copied `stmt_let_ty` to get the starting point for this function and didn't remove this unused argument. I will fix it shortly.